.github/workflows/codeql.yml

```diff
@@ -39,6 +39,7 @@ jobs:
           proxy.golang.org:443
           repo.maven.apache.org:443
           storage.googleapis.com:443
+          uploads.github.com:443
```

Comment:
nit: *.github.com since we already have api.github.com too? wdyt of using more wildcards or does that defeat some of the security benefits?